ragna/deploy/_ui/left_sidebar.py

```diff
@@ -25,12 +25,12 @@ def __init__(self, api_wrapper, **params):
             on_error=lambda x: print(f"error sync on {x}"),
         )
 
-    def trigger_on_click_new_chat(self, event):
+    async def trigger_on_click_new_chat(self, event):
```

Comment:
`on_click_new_chat` now needs to be an async function (see `main_page.py` and `self.open_modal` for details).